frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -686,7 +847,37 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
         <div className="co-m-pane__body-group">
           <div className="row">
             <div className="col-sm-6">
-              <ResourceSummary resource={props.obj} />
+              <ResourceSummary resource={props.obj}>
+                <dt>
+                  <Popover
+                    headerContent={<div>Managed Namespaces</div>}
+                    bodyContent={
+                      <LinkifyExternal>
```

Comment:
No reason for `LinkifyExternal` here since we know what the content is (and it has no links).